crates/codegen/runtime/cargo/src/runtime/language.rs.jinja2

```diff
@@ -228,7 +228,7 @@ impl Lexer for Language {
                             if kw_scan == KeywordScan::Absent {
                                 input.set_position(save);
 
-                                // TODO(#638): Don't allocate a string here
+                                // TODO(#723): Don't allocate a string here
```

Comment:
#723 was merged in January. Is this intended?
I suggest using an active/open issue to track `TODO` items, so that we don't forget about them?

Reply:
It is intended and I don't know why this issue was mentioned for this snippet in the first place. This logic was a part of #723 and it was meant to emphasize a possible low hanging fruit in the future; it was never a part of #638 work and wasn't supposed to be tackled as part of the remaining v2 migration.

I'd say `TODO(#` is easily greppable and this item is small enough that it might probably not warrant a dedicated issue but if you think otherwise then I'm happy to open an issue :thumbsup:

Reply:
> was never a part of https://github.com/NomicFoundation/slang/issues/638 work and wasn't supposed to be tackled as part of the remaining v2 migration.

Sounds good.

> this item is small enough that it might probably not warrant a dedicated issue

I suggest creating an issue and attaching it here still, as it affects the product perf, not just internal build/codegen. Please feel free to assign it to me, and I can investigate it as part of my upcoming perf work.

Reply:
Opened https://github.com/NomicFoundation/slang/issues/1001